elfo-network/src/socket/raw/mod.rs

```diff
@@ -95,6 +96,25 @@ impl AsyncWrite for OwnedWriteHalf {
     }
 }
 
+impl OwnedWriteHalf {
+    /// Collects and reports transport-specific metrics.
+    pub(crate) fn collect_transport_specific_metrics(&self) {
+        match self {
```

Comment:
It's the second usage of delegation (the first one is `is_write_vectored`), let's create a macro similar to `delegate_call` (which should be renamed)

Reply:
we cant reuse `delegate_call` here because neither uds nor Turmoil has metrics reporting.

Reply:
It was about creating empty impls of `collect_transport_specific_metrics`.

Anyway, lgtm